### DISCUSSION TITLE
Welcome to vajra Discussions!

### DISCUSSION THREAD

**Discussion by r3curs1v3-pr0xy** [r3curs1v3-pr0xy/vajra#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below.

**Comment by 0xN4rsi-421** [comment]
can any one help me with it 
![Capture](https://user-images.githubusercontent.com/51074238/114411526-e05f3600-9bc9-11eb-9e60-69490611ba51.PNG)

**Comment by r3curs1v3-pr0xy** [comment]
Install the latest version of docker-compose.  https://docs.docker.com/compose/install/

**Reply by 0xN4rsi-421** [reply]
![image](https://user-images.githubusercontent.com/51074238/114582402-2e456e00-9c9e-11eb-9071-ad2e13c0641a.png)
getting this error

**Comment by r3curs1v3-pr0xy** [comment]
Actually this is not an error. That's a couchdb container logs.

**Reply by 0xN4rsi-421** [reply]
chttpd_auth_cache changes listener died because the _users database does not exist. Create the database to silence this notice.

how can i create it

**Comment by 0xN4rsi-421** [comment]
How can i reslove it it asking to create couchdb for users

**Comment by r3curs1v3-pr0xy** [comment]
Add any target to the scan. It'll automatically create a database.

**Comment by abhay2510kr** [comment]
can u please tell me how to run these tools i have installed all the thing every thing but how I can run I am going on a localhost but I am not able to get anything 127.0.0.1

**Comment by 0xN4rsi-421** [comment]
Try to build it with docker

**Reply by abhay2510kr** [reply]
Ok

**Reply by abhay2510kr** [reply]
Let me scanner
